### ISSUE TITLE
Bogus flag testing

### ISSUE THREAD

**Issue by mathstuf** [trentforkert/cmake#11]
GDC-on-Linux really shouldn't be testing these MSVC flags:

```
-- The D compiler identification is GNU 2.64
cc: error: /nologo: No such file or directory
cc: error: /showIncludes: No such file or directory
```

I'm currently debugging to try and see where they're coming from.

**Comment by trentforkert** [comment]
Fixed? I pushed a change to address this (6ae82a9514bffca38ca7545de9641603bad3e6d1), but can't reproduce it here with or without this change.

**Comment by mathstuf** [comment]
I haven't seen this in a long time. Closing.
